static-assets/components/cstudio-forms/data-sources/img-desktop-upload.js

```diff
@@ -53,7 +53,7 @@ YAHOO.extend(CStudioForms.Datasources.ImgDesktopUpload, CStudioForms.CStudioForm
 		
 		var callback = { 
 			success: function(imageData) {
-                var topWin = window.parent.Self;
+                var topWin = window.parent.self;
```

Comment:
This refers to WCMRootContent or so, please call it it explicitly.

Reply:
Done